src/editors/query/query.url.tsx

```diff
@@ -158,10 +193,11 @@ const Body = ({ query, onChange, onRunQuery }: { query: InfinityQuery; onChange:
   }
   const placeholderGraphQLQuery = `{ query : { }}`;
   const placeholderGraphQLVariables = `{ }`;
+  const doesBodyAllowed = ['POST', 'PUT', 'PATCH', 'DELETE'].includes(query.url_options?.method);
```

Comment:
Can we add the same comment that we have in `pkg/infinity/client.go`  here too?

> // according to https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods, GET method should not contain request body

Reply:
Done in https://github.com/grafana/grafana-infinity-datasource/pull/1110/commits/5616500bed3fb58086c93adb77a0205f75ffe1a7